### ISSUE TITLE
PR: IfcSpatialContainer doesn't work properly 

### ISSUE THREAD

**Issue by Mairh93** [Autodesk/revit-ifc#695]
### Problem Description

Dear all,

I am working in Revit 2023 and IFC Version 23.3.1.0. I added the parameter IFCSpatialContainer from the Autodesk's .txt file. 

In Revit I assigned to my elements two different levels. When I export in IFC the elements are not in the levels that I assigned. 

See images below:

![UP level](https://github.com/Autodesk/revit-ifc/assets/27021809/d1cc1688-40d9-4868-b860-c80e68cb2316)
![down level ](https://github.com/Autodesk/revit-ifc/assets/27021809/30441adb-62f7-4dbb-b1a1-ee9e48ab2669)
![BIMVision](https://github.com/Autodesk/revit-ifc/assets/27021809/dd30c148-a8e3-43c8-ba6b-27735337468d)

Thank you in advance!

### Revit Version

2023.1.x

### IFC for Revit Addon Version

Other / None

### Windows Version

11 22H2

**Comment by Mairh93** [comment]
Yes I used the IfcName parameter in order to 'rename' the level in the ifc exportation. The puprose of the ifcSpatialContainer parameter is to assign the elements in the level that I need.

**Comment by AngelVelezSosa** [comment]
IfcSpatialContainer may be based on the *internal* (not IFC) name of the Level.  So instead of deleting IfcName, can you set IfcSpatialContainer to the "real" name of the level?  I don't think one is more correct than the other, but I wouldn't want to toggle the behavior since it would break the other half of the people (assuming I am correct in my first line of assumption).

**Comment by ArtemHnatko** [comment]
Hi @Mairh93,

As we haven't received the requested sample model or confirmation regarding the tests (matching internal Level names vs. overrides), we are closing this ticket for now to keep our tracking system up to date.

The interaction between these parameters can be complex, and without a reproducible case, we cannot determine if this requires a code change or remains a known limitation of the current IFC export logic.

If you are still encountering this issue or can provide a minimal RVT file for testing, please feel free to reopen this ticket at any time. We would be happy to take another look.
